fastlane/Fastfile

```diff
@@ -428,4 +428,44 @@ platform :ios do
 
   end
 
+  desc "Generate release podspecs"
+  lane :generate_release_podspecs do |options|
+
+    xcframeworks_folder_path = options[:xcframeworks_folder_path]
+    podspec_name = options[:podspec_name] #e.g: GiniBankSDK or GiniBankSDKPinning
+    podspecs_repo_sdk_folder_path = options[:podspecs_repo_sdk_folder_path]
+
+    # Get the latest Git tag
+    latest_tag = sh("git describe --tags --abbrev=0").strip
+
+    # Extract the version from the tag (assuming the tag format is "GiniBankSDK;X.X.X")
+    version_folder = latest_tag.match(/;(\d+\.\d+\.\d+)/).captures.first
+    
+    # Step 1: Compress the XCFrameworks into a zip file
+    zip_name = "#{podspec_name}-XCFrameworks.zip" #e.g GiniBankSDK or GiniBankSDKPinning
+    sh "cd #{xcframeworks_folder_path} && zip -r #{zip_name} ."
+    
+    # Step 2: Create a subfolder named after the version
+    sdk_folder = "#{xcframeworks_folder_path}/#{version_folder}"
+    sh "mkdir -p #{sdk_folder}"
+
+    # Step 3: Copy the zip file and podspec into the new folder
+    sh "cp #{xcframeworks_folder_path}/#{zip_name} #{sdk_folder}/"
+    sh "cp ../BankSDK/#{podspec_name}/Pod/#{podspec_name}.podspec #{sdk_folder}/" #e.g: BankSDK/GiniBankSDK or BankSDK/GiniBankSDKPinning
```

Comment:
[Request] "BankSDK" is hardcoded in the path. The path of the template `.podspec` file should be passed in as a lane parameter for maximum flexibility:
```suggestion
    sh "cp #{template_podspec_path} #{sdk_folder}/"
```